### PR TITLE
Dev summary 07 31

### PR DESCRIPTION
**Pull request by blackboxsw**
Chained pull request adding just the scrubbed 2018-07-31.md content. It includes the  prerequisite pull https://github.com/CanonicalLtd/server-dev-summary/pull/1.

### CODE REVIEW

File: doc/2018/2018-07-31.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Hello Ubuntu Server
+
+The purpose of this communication is to provide a status update and
+highlights for any interesting subjects from the Ubuntu Server Team. If
+you would like to reach the server team, you can find us at
+the #ubuntu-server channel on Freenode. Alternatively, you can sign up
+and use the Ubuntu Server Team [mailing list](https://lists.ubuntu.com/mailman/listinfo/ubuntu-server).
+
+## Spotlight:
```
Comment (powersj) [inline doc/2018/2018-07-31.md]: This needs a title... "## Spotlight: Ubuntu Bionic 18.04.1 and new ISO"

File: doc/2018/2018-07-31.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+The Ubuntu Server Installer (subiquity-based) has been released in Bionic
```
Comment (powersj) [inline doc/2018/2018-07-31.md]: "Along with the first point release of Ubuntu Bionic (18.04.1), the Ubuntu Server installer was updated."

File: doc/2018/2018-07-31.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Ubuntu Server ISOs. It now carries additional configuration support for:
+ - LVM
+ - RAID
+ - VLAN
+ - Bonds.
```
Comment (powersj) [inline doc/2018/2018-07-31.md]: no period

File: doc/2018/2018-07-31.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+There is a [call for testing](https://lists.ubuntu.com/archives/ubuntu-server/2018-July/007725.html) for these features in Ubuntu.  [Josh Powers](https://github.com/powersj) on Canonical's ubuntu server team has [a blog post explaining with details about the new installer features](https://powersj.github.io/post/ubuntu-bionic-subiquity-updated/).
+
```
Comment (powersj) [inline doc/2018/2018-07-31.md]: Can you change the first URL to:
https://powersj.io

And my blog URL to 
https://powersj.io/post/ubuntu-bionic-subiquity-updated/

File: doc/2018/2018-07-31.md (hunk 2 above)
Comment (blackboxsw) [reply doc/2018/2018-07-31.md]: replaced.

File: doc/2018/2018-07-31.md (hunk 3 above)
Comment (blackboxsw) [reply doc/2018/2018-07-31.md]: done

File: doc/2018/2018-07-31.md (hunk 4 above)
Comment (blackboxsw) [reply doc/2018/2018-07-31.md]: done. thanks

**Review by blackboxsw — COMMENTED** [review]
just pushed fixes. thanks josh

**Pull request merged**